source/Depreciation.cpp

```diff
@@ -226,18 +229,20 @@ int64_t Depreciation::Value(const vector<shared_ptr<Ship>> &fleet, int day) cons
 	for(const auto &it : shipCount)
 		value += Value(it.first, day, it.second);
 	for(const auto &it : outfitCount)
-		value += Value(it.first, day, it.second);
+		// Because this function is only called in flight it makes no sense to use any local modifiers to outfit prices.
+		// Also the result from this should not fluctuate with local conditions, given it is used to calculate bank allowance etc.
```

Comment:
The biggest thing is it makes no sense to make the player's fleet value fluctuate based on what system he is in.
I'll clarify it, whether we are in flight or not matters not, what does is that how much the bank will give you depends on it, and it can't change in every system.

Reply:
This is sort of a realism vs gameplay thing: I'd fully expect Network to fluctuate based on location and have access to different sized loans based on where I am. 

Ex. A dirbelt world might have zero understanding or appreciation for having a ship filled with alien tech, and thus be unwilling to accept it as collateral for a loan. On the flip side, the Remnant would have a fairly good understanding of what that tech is worth and be willing to accept it as such. 

Of course, that assumes that the banks are different, not one huge pan-galactic entity that we currently have in-game...

Impact:

My thought is that this is a lore/setting choice for us. Establishing Networth and loan values as a constant that doesn't change based on where the player is reinforces the idea that The Bank is a pan-galactic entity.

Having Networth and ripple effect vary based on location moves it towards feeling like each planet has their own Bank and they merely work together to collect fines/payments/etc. In real life loan shopping is very much a thing. Different banks in different countries and cities have different tolerances for risk and different appreciation of the collateral one brings to the table. It shouldn't be surprising (and in fact, could add some interesting depth to the game) if one can get a better loan from one bank than from another.

Given that Networth doesn't have much impact outside of financial calculations, I think I would prefer the later. Anything we can do to give the impression that the bank is *NOT* a pan-galactic entity is, I think, a good thing.

Reply:
It might give that impression, but it could lead to players either not understanding or abusing of it, and going in certain systems to get absurd credits. I don't personally mind it but there should indeed be a discussion about it.

Performance wise it's worse, and there is a technical problem: what do we do if there are multiple outfitters in the system? To fix both issues (caus who cares about performances once you're landed) I propose checking if the player is landed as the day passes, if he is then use that local outfitter, otherwise we use general values?

Reply:
Well that's what I did, look if the player is landed and if he is use the local values of the outfitter.

Reply:
I think our players can probably figure it out, especially if it is mentioned somewhere. (It'd be great to be able to trigger missions in the outfitter, shipyard, and bank panels like we currently do in the starport. Then we'd just have a mission pop up the first time they go to the bank)

That being said, those "abuses" are kind of core parts of our economic and financial system (both real life and fictional). Basically it's the same thing as using bank loans to cover commodity trading, except the commodity is a tangible thing that we actual use (and has depreciation attached to it, although that probably won't really have much effect). They're just a little more visible because we, the pilot, have already paid the opportunity cost to access it (whereas in real life most people *haven't* paid them)

As far as that goes, it might make the most sense to only update the networth and values on landing. That reduces the frequency that this comes up, and resolves the complexity about situations where there are more than one outfitter in the system.

How much of a performance impact are we talking about? Given that this would be happening on landing, which is a transition to a situation that is fairly static, I would expect a little bit of delay to be acceptable and pretty much concealed by the transition. And it should only need to trigger once per landing, right? So is this increased performance impact noticeable? Are we talking milliseconds, or something larger?

Reply:
I very much like the abuses being possible! I like minmaxing stuff

> As far as that goes, it might make the most sense to only update the networth and values on landing. That reduces the frequency that this comes up, and resolves the complexity about situations where there are more than one outfitter in the system.

Idk why its being updated at other times, I can't find any usage of older worths in the bank, but I try not to touch what I don't grasp so I'll ask smbd who knows about it in the discord

About the performance impact, as I said earlier, 
> If we are to do it, to fix those two issues (caus who cares about performances once you're landed) I propose checking if the player is landed as the day passes, if he is then use that local outfitter, otherwise we use general values?

Which is what I did so the impact is negligible.

Reply:
I think I'll be opening a refactoring of missions soon, with some new possibilities like starting a mission in a system or outfitter and whatnot.
I didn't do it yet because it seems like the reviewers are always busy

Reply:
Net worth is used for calculating bribes so we have to calculate it whenever a day passes, in case the player gets a big ship captured for instance.

Reply:
I've always thought it weird that pirates somehow know exactly how much you are worth and charge a uniform rate based on that.

Reply:
They should only charge based on the ships they see in the system indeed